ray-operator/test/e2eupgrade/rayservice_operator_upgrade_test.go

```diff
@@ -76,15 +82,32 @@ func TestZeroDowntimeUpgradeAfterOperatorUpgrade(t *testing.T) {
 	g.Expect(endpoints.Subsets[0].Addresses).To(HaveLen(1))
 
 	// Upgrade KubeRay operator to latest version and replace CRDs
-	test.T().Logf("Upgrading the KubeRay operator to the latest release")
-	cmd := exec.Command("kubectl", "replace", "-k", fmt.Sprintf("github.com/ray-project/kuberay/ray-operator/config/crd?ref=%s", upgradeVersion)) //nolint:gosec // required for upgrade
-	err = cmd.Run()
-	g.Expect(err).NotTo(HaveOccurred())
-	g.Eventually(cmd, TestTimeoutShort).Should(WithTransform(ProcessStateSuccess, BeTrue()))
-	cmd = exec.Command("helm", "upgrade", "kuberay-operator", "kuberay/kuberay-operator", "--version", upgradeVersion)
-	err = cmd.Run()
-	g.Expect(err).NotTo(HaveOccurred())
-	g.Eventually(cmd, TestTimeoutShort).Should(WithTransform(ProcessStateSuccess, BeTrue()))
+	test.T().Logf("Upgrading the KubeRay operator to %s", upgradeVersion)
+	if upgradeVersion == nightlyUpgradeImageVersion {
```

Comment:
Can you use the Helm chart on the same commit instead of `kubectl set image`? We can also add the upgrade test from the latest release to the nightly commit in the CI in a follow up PR.

Reply:
I don't think helm can be used, because the `nightly` isn't available there (if I understand your question correctly)

Reply:
@kevin85421 are you ok with leaving the code as is and merging this PR if you are happy?